era5_cloud/download_era5.py

```diff
@@ -0,0 +1,39 @@
+import cdsapi
```

Comment:
Can you add instructions/links to set this up?

Reply:
I added the instruction in the README